Mint/README.md

```diff
@@ -1,35 +1,149 @@
+- Works out of the box with full multimedia support (requires very little effort to setup).
+- Very stable and low maintenance.
+- It's based on Ubuntu which gives the user a large number of available packages (around 30,000).
+- It's a very beginner friendly distro.
+- provides great stability with software packages and balances that with a hardware enablement stack to give newer computers a fighting chance at working with Linux.
+- Beautiful Desktop
+- Drivers, Multimedia, Tools Support
+-  An Open-Source Philosophy
+- A Welcoming Community
+
+For Extra Knowledge You cand find details [here](https://www.fossmint.com/reasons-to-use-linux-mint/)
+
 
-``(Stability , Security , Performance , Compatible apps , Hardware/System requirements, Technical support of the distro's maintainers) and any other points you find``
 ## It's Cons
-``-> Add distro cons relating to: ``
+|          Related TO          |                                                                                                                   Cons                                                                                                                    |
+|:----------------------------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|           Security           |                                                        The Linux Mint project does not provide security advice and has been criticized for not doing so on at least one occasion.                                                         |
+|         Performance          |           Though Linux Mint may seem a little faster to use daily, but on older hardware, it will feel faster, while Ubuntu seems to work more better as the software ages. Mint becomes even faster when it runs, like Ubuntu.           |
+|       Compatible apps        |                                                  Some of the most popular Windows apps that are compatible with Linux Mint include Microsoft Office, Skype, Adobe Photoshop, and iTunes                                                   |
+| Hardware/System requirements |                                                                                      2 GB of RAM , 20 GB of hard drive space , 1024 x 768 resolution                                                                                      |
+|      Technical support       |                               Does not have support for a large number of popular applications and drivers. If you need these, you should consider other distributions such as Ubuntu or Linux Mint itself.                               |
+
+## Who is it made for? (target users) 
+
+-I think that Mint's target user is someone who comes from a windows background.
+
+-Linux Mint does not have background processes that eat up precious system resources!
```

Comment:
No need to add this here, since its place is the pros, so you can just remove it

Reply:
My bad since I specified all 4 lines, I meant to **only remove this part** "Linux Mint does not have background processes that eat up precious system resources!" and **leave this one** "Mint's target user is someone who comes from a windows background."